firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHttpClient.java

```diff
@@ -182,9 +182,15 @@ FetchResponse fetch(
       String lastFetchETag,
       Map<String, String> customHeaders,
       Long firstOpenTime,
-      Date currentTime)
+      Date currentTime,
+      Boolean excludeEtagHeaderForRealtime)
```

Comment:
Instead of calling this with `false` when that's the default, we could be more direct/clear in the callers (and set a good default option when using fetch) by creating an overload and one new method for the "exclude etag" case. That'd also make it easier to turn the double negative around for the default case:

```
FetchResponse fetch(...) {
  fetch(..., /* withEtag */ true);
}

FetchResponse fetchWithoutEtag(...) {
  fetch(..., /* withEtag */ false);
}

private FetchResponse fetch(..., includeEtag) {
  /* existing code + etag condition */
}